monai/bundle/utils.py

```diff
@@ -107,6 +107,7 @@
         "tracking_uri": "$@output_dir + '/mlruns'",
         "experiment_name": "monai_experiment",
         "run_name": None,
+        "execute_config": "placeholder, may fill at runtime",
         "is_not_rank0": (
```

Comment:
Hi @Nic-Ma ,
When running the bundle, in the init function of mlflow handler the artifacts var input value if `None`.

Thanks,
Bin